src/api/topics/routes.py

```diff
@@ -34,7 +34,7 @@ def get() -> list[dict[str, Any]] | None:
         parsed_args = parser.parse_args()
         topics = Topic.get_topics(
             sorting=parsed_args.get("order_by"),
-            author_ids=parsed_args.get("author_ids"),
+            author_ids=parsed_args.get("author_id"),
```

Comment:
Why is this line changed? Doesn't it work with `.get("author_ids"` ?

Reply:
yes, it doesn't. we have an argument "author_id" defined in parser, but we try to get "author_ids" from parsed_args.

Reply:
Ok, seems like it's a bug, so created a new ticket for that: https://github.com/fenya123/forum123/issues/144